packages/react-dom/src/__tests__/ReactDOMFizzServer-test.js

```diff
@@ -2872,4 +2872,173 @@ describe('ReactDOMFizzServer', () => {
       expect(window.__test_outlet).toBe(1);
     });
   });
+
+  // @gate experimental && enableClientRenderFallbackOnTextMismatch && enableClientRenderFallbackOnHydrationMismatch
```

Comment:
Both flags just landed so I think you can rebase and remove them here.

Reply:
well that was fast :)